prime-router/build.gradle.kts

```diff
@@ -509,6 +509,7 @@ flyway {
 
 // Database code generation configuration
 jooq {
+    version.set("3.15.4")
```

Comment:
This is a patch version bump from the default of `3.15.1` in order to include https://github.com/jOOQ/jOOQ/issues/12508